app/lib/io/zipUtil.js

```diff
@@ -0,0 +1,44 @@
+define([
+    'jszip'
+  , 'io/readDirRecursive'
+  , 'metapolator/errors'
+], function(
+    JSZip
+  , readDirRecursive
+  , errors
+) {
+    "use strict";
+
+    var NotImplementedError = errors.NotImplemented;
+
+    var unpack = function(async, zipData, io, targetPath){
+        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+    };
+
+    var encode = function(async, io, sourcePath, dataType){
+        dataType = dataType || 'base64';
+
+        if(async)
+            throw new NotImplementedError('Asynchronous execution is not yet implemented');
+
+        var zip = new JSZip(),
+            files = readDirRecursive(false, io, sourcePath),
+            i
+            ;
+        for (i=0; i<files.length; i++){
+            var file = files[i],
+                relative_path = file.split(sourcePath)[1],
+                data = io.readFile(false, file)
+                ;
+            zip.file(relative_path, data, {binary:true});
+        }
+
+        var zipped_data = zip.generate({type:dataType});
+        return new Buffer(zipped_data, dataType);
+    };
```

Comment:
no need for `return new Buffer(...`
just `return zip.generate({type:dataType});`